doc/manual/src/contributing/hacking.md

```diff
@@ -282,3 +282,45 @@ See also the [format documentation](https://github.com/haskell/cabal/blob/master
 
 Releases have a precomputed `rl-MAJOR.MINOR.md`, and no `rl-next.md`.
 Set `buildUnreleasedNotes = true;` in `flake.nix` to build the release notes on the fly.
+
+## Branches
+
+- `master`
```

Comment:
Maybe link the branch names to GitHub? There are even search queries you can link.